### PR TITLE
Fix phpstan

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this should be BC.

### CODE REVIEW

File: src/Admin/AbstractAdmin.php [hunk 1]
```diff
@@ -103,6 +103,7 @@ abstract class AbstractAdmin extends AbstractTaggedAdmin implements AdminInterfa
     private const MASK_OF_ACTIONS_USING_OBJECT = self::MASK_OF_ACTION_SHOW | self::MASK_OF_ACTION_EDIT | self::MASK_OF_ACTION_HISTORY | self::MASK_OF_ACTION_ACL;
 
     private const DEFAULT_LIST_PER_PAGE_RESULTS = 25;
+
```
Comment (jordisala1991) [inline src/Admin/AbstractAdmin.php]:
```suggestion
```

File: src/Admin/AbstractAdmin.php [hunk 2]
```diff
@@ -1655,7 +1655,7 @@ final public function supportsPreviewMode(): bool
     /**
      * Returns predefined per page options.
      *
-     * @return list<int>
+     * @return array<int>
```
Comment (jordisala1991) [inline src/Admin/AbstractAdmin.php:1658]: This is the only one I have doubts, why we were using list? Phpstan complains with:

```
 ------ --------------------------------------------------------------------------------------------------------
  Line   src/Admin/AbstractAdmin.php
 ------ --------------------------------------------------------------------------------------------------------
  1672   Method Sonata\AdminBundle\Admin\AbstractAdmin::getPerPageOptions() should return list<int> but returns
         non-empty-array<0|1|2|3|4|5, int>.
```

Comment (VincentLanglet) [reply src/Admin/AbstractAdmin.php:1658]: Only usage is `{% for per_page in admin.getperpageoptions %}` and
`return \in_array($perPage, $this->getPerPageOptions(), true);`
So it doesn't require to be a list.

I have one more question:
Why this method is not in the interface but is public and used in the twig ?

Can you add it (with @method) ?

**Review by VincentLanglet — APPROVED** [review]

File: src/Admin/AbstractAdmin.php (hunk 2 above)
Comment (VincentLanglet) [reply src/Admin/AbstractAdmin.php:1658]: Anyway can be done in another pR

**Pull request merged**